webrender/src/device.rs

```diff
@@ -2723,45 +2608,48 @@ impl<B: hal::Backend> Device<B> {
 
         assert_eq!(texture.fbo_ids.len(), 0);
 
-        let (depth_rb, allocate_depth) = match texture.depth_rb {
-            Some(rbo) => (rbo, is_resized || !rt_info.has_depth),
-            None if rt_info.has_depth => {
-                let depth_rb = self.generate_rbo_id();
-                texture.depth_rb = Some(depth_rb);
-                (depth_rb, true)
-            },
-            None => (RBOId(0), false),
-        };
+        // RGBAF32 images aren't used as render targets, thus they don't need to have associated FBOs.
```

Comment:
we shouldn't make this decision based on the format - there is already `render_target`, which being `None` signifies that no FBO is needed